locales/pl/base.json

```diff
@@ -0,0 +1,233 @@
+    "save": "Zapisz",
+    "cancel": "Anuluj",
+    "delete": "Usuń",
+    "entry-inputs-empty-info": "Podaj przynajmniej tytuł.",
+    "entry-title-empty-info": "Wpisz tytuł.",
+    "custom-fields-label-empty-info": "Zapomniałeś o uzupełnieniu etykiety dla przynajmniej jednego dodatkowego pola.",
+    "are-you-sure-question": "Jesteś pewny?",
+    "displaying-entries": "Liczba wpisów: {{count}}",
+    "no-value": "Brak wartości"
+  },
+  "entry-menu": {
+    "copy-to-clipboard": "Kopiuj do schowka",
+    "username": "Nazwa użytkownika",
+    "password": "Hasło",
+    "open-url-in-browser": "Otwórz URL w przeglądarce",
+    "move-to-group": "Przenieś do grupy",
+    "move-to-trash": "Przenieś do śmietnika",
+    "delete-permanently": "Usuń na stałe"
+  },
+  "error": {
+    "archive-not-found": "Nie znaleziono źródła dla archiwum.",
+    "authentication-failed": "Błąd autentykacji.",
+    "unknown": "Wystąpił nieznany błąd",
+    "passwords-dont-match": "Hasła nie zgadzają się.",
+    "group-not-found": "Nie znaleziono grupy.",
+    "sort-definition-not-found": "Nie znaleziono sposobu sortowania",
+    "insufficient-data-provided": "Niewystarczające dane do sortowania",
+    "check-for-update": "Podczas aktualizacji wystąpił błąd. Spróbuj jeszcze raz później.",
+    "invalid-import-type-requested": "Nieprawidłowy typ importu",
+    "open-file-for-importing": "Import nie powinien być uruchomiony bez głównego okna",
+    "entry-not-found": "Nie znaleziono wpisu",
+    "dropbox-connection-failed-info": "Połączenie z serwerem Dropbox nie powiodło się. Spróbuj jeszcze raz później",
+    "webdav-connection-failed-info": "Połączenie z {{endpoint}} nie powiodło się. Sprawdź swoje dane i spróbuj jeszcze raz."
+  },
+  "group": {
+    "new-group": "Nowa grupa",
+    "untitled": "Bez tytułu"
+  },
+  "group-menu": {
+    "new-group": "Nowa grupa",
+    "add-group": "Dodaj grupę",
+    "move-to-root": "Przenieś do głównego poziomu",
+    "move-to-group": "Przenieś do grupy",
+    "rename": "Zmień nazwę",
+    "delete": "Usuń",
+    "move-to-group-custom": "Przenieś do {{title}}",
+    "empty-trash": "Opróżnij śmietnik",
+    "empty-trash-question": "Czy na pewno chcesz opróżnić śmietnik?",
+    "title-asc": "Tytuł: Rosnąco",
+    "title-desc": "Tytuł: Malejąco"
+  },
+  "intro": {
+    "welcome-title": "Witaj w Buttercup.",
+    "welcome-back-title": "Witaj ponownie w Buttercup.",
+    "welcome-caption": "Nie dodałeś jeszcze żandego archiwum. Czy chcesz dodać je teraz?",
+    "unlock-archive": "Odblokuj archiwum żeby rozpocząć ({{os}})."
+  },
+  "intro-menu": {
+    "open-archive-file": "Otwórz plik archiwum",
+    "new-archive-file": "Nowy plik archiwum",
+    "connect-cloud-sources": "Podłącz źródło w chmurze"
+  },
+  "password-dialog": {
+    "master-password": "Hasło główne",
+    "new-password": "Nowe hasło",
+    "password": "Hasło",
+    "confirm-password": "Potwierdź hasło",
+    "confirm": "Potwierdź",
+    "nevermind": "Nie ważne"
```

Comment:
I'm not sure about this one. If someone could show me the password dialog maybe I could adjust it more.